From 462aa0c53397ec5bf78e3e7f68aa8a3ca300f4ba Mon Sep 17 00:00:00 2001
From: Selvam Sathappan Periakaruppan <speriaka@codeaurora.org>
Date: Tue, 24 Mar 2020 19:09:38 +0530
Subject: [PATCH 5/8] clk: qcom: ipq8074: Fix gcc_snoc_bus_timeout_ahb_clk
 offset

By default, the ipq8074 V2 clks are provided in the gcc driver.
Updating the gcc_snoc_bus_timeout_ahb_clk offsets also as needed
in ipq8074 V2.

Change-Id: I5a6e98d002f5c3354a804e55dd9ebb1f83f7f974
Signed-off-by: Selvam Sathappan Periakaruppan <speriaka@codeaurora.org>
---
 drivers/clk/qcom/gcc-ipq8074.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/clk/qcom/gcc-ipq8074.c
+++ b/drivers/clk/qcom/gcc-ipq8074.c
@@ -4488,10 +4488,10 @@ static struct clk_branch gcc_pcie0_axi_s
 };
 
 static struct clk_branch gcc_snoc_bus_timeout2_ahb_clk = {
-	.halt_reg = 0x4700c,
+	.halt_reg = 0x47014,
 	.halt_bit = 31,
 	.clkr = {
-		.enable_reg = 0x4700c,
+		.enable_reg = 0x47014,
 		.enable_mask = BIT(0),
 		.hw.init = &(struct clk_init_data){
 			.name = "gcc_snoc_bus_timeout2_ahb_clk",
@@ -4506,10 +4506,10 @@ static struct clk_branch gcc_snoc_bus_ti
 };
 
 static struct clk_branch gcc_snoc_bus_timeout3_ahb_clk = {
-	.halt_reg = 0x47014,
+	.halt_reg = 0x4701C,
 	.halt_bit = 31,
 	.clkr = {
-		.enable_reg = 0x47014,
+		.enable_reg = 0x4701C,
 		.enable_mask = BIT(0),
 		.hw.init = &(struct clk_init_data){
 			.name = "gcc_snoc_bus_timeout3_ahb_clk",
